docs/content/dev/setup/frontend.md

```diff
@@ -52,3 +52,10 @@ You can inspect the generated production-bundle files to look for optimization i
 
 1. run `yarn analyze`. This will overwrite any existing builds!
 2. Open the generated `.next/server/analyze/server.html` in your favourite browser
+
+## Enable Logging in Production
```

Comment:
That's not quite right. Logging is available in production. The only thing that is disabled is debug logging.

Reply:
@mrdrogdrog I have changed it to "Enable Debug Logging in Production"